src/requestLogger.js

```diff
@@ -27,6 +29,14 @@ class RequestLogger {
     this.invocationId = uuid.v4();
   }
 
+  /**
+   * Allows to set the static data. It will override static data specified with the configuration at construction time.
+   * @param {Object} staticData Any static data that are assigned to every log message. Typical might be an environment parameter or version number.
+   */
+  setStaticData(staticData) {
```

Comment:
I'm almost thinking this belongs as the parameter to `startInvocation` and won't be needed here or in the constructor, but it would be good to better understand the use case.

Reply:
Makes sense. I've made it part of that method.

The unit test is now a mess. I didn't find an easier method to compare the log result since it now contains the invocationId. An alternative would be to reset it back to NULL, but it's a useful addition to the test anyways. Thoughts?

Reply:
Whatever, someone can complain later.  It's good and we can come back to that if we figure out a better solution.